src/tests/suite/daos_test_common.c

```diff
@@ -110,7 +110,7 @@ test_setup_pool_create(void **state, struct test_pool *ipool,
 	}
 out:
 	/** broadcast pool create result */
-	if (arg->multi_rank) {
+	if (arg->multi_rank && outpool->svc != NULL) {
```

Comment:
i thought the svcl was removed. my only concern is that if it's not required, we don't bcast the pool uuid in that case.

Reply:
Hmm. Good point. I was focused on fixing the Coverity defect where `outpool->svc` could be NULL if the number of service ranks is zero, but I guess maybe it would be better to ask if we should even have `outpool->svc` anymore.

What happens if we remove the broadcast of the service ranks? I'm assuming that this multi_rank path isn't taken in CI, so I'm not sure how to test it. Maybe we should just fix the immediate issue for 1.2 and then revisit this as part of a cleanup?

Reply:
I could change this so that the NULL check for `outpool->svc` is done before the broadcast. That would probably be clearer anyhow.

Reply:
i agree that this would be the simplest / safest fix for the coverity issue

Reply:
Done.